src/ngraph/frontend/onnx_import/wrappers/node.hpp

```diff
@@ -16,8 +16,6 @@
 
 #pragma once
 
-#include <algorithm>
```

Comment:
Please revert the change. This header file uses `algoriths`explicitly. The header files MUST be self-contained (this is the rule). There's no guarantee that any of header file will provide the rest of header files you depend on implicitly (you should never think like that).